kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -101,16 +106,45 @@ public ClientConfiguration newClientConfiguration(S3SinkConnectorConfig config)
     clientConfiguration.withUserAgentPrefix(version);
     if (StringUtils.isNotBlank(config.getString(S3_PROXY_URL_CONFIG))) {
       S3ProxyConfig proxyConfig = new S3ProxyConfig(config);
+      RetryPolicy fullJitterPolicy = newRetryPolicy(config);
```

Comment:
nit: Could also be called in place in the `withRetryPolicy` especially if the method is renamed.